apps/mosaico/src/Article/Article.purs

```diff
@@ -276,6 +282,30 @@ render imageComponent props =
         , onTagClick: props.onTagClick
         })
 
+    renderAdvertorialTeaser article =
+      let
+        imgSrc =
+          maybe (fallbackImage props.paper)
+            ((_ <> "&function=hardcrop&width=200&height=200&q=90") <<< _.url)
```

Comment:
We probably shouldn't add these query params, as they work only with the soon-to-be-deprecated imengine. Instead, we could try the `thumb` of the image, which should be big enough for this purpose.